File: Two_Sum/two_sum.java
package Two_Sum;


class Main{
    public static void main(String[] args) {
        try {
            int[] nums = {1, 2, 3, 4};
            int[] indices = {1, 5};

            int n1 = nums[indices[0]];
            int n2 = nums[indices[1]];

            System.out.println();
            System.out.println(n1 + n2);

        } catch (Exception e) {
            System.out.println();
            System.out.println("Erro: " + e.getMessage());
        }
    }
}

File: Two_Sum/two_sum.php
<?php

try{

    $indices = [1, 2];
    $nums = [1, 2, 3, 4];

    if(!isset($nums[$indices[0]]) || !isset($nums[$indices[1]])){
        throw new Exception("As chaves estão vazias");
    }

    $n1 = $nums[$indices[0]];
    $n2 = $nums[$indices[1]];

    echo $n1 + $n2;

} catch (Exception $e){
    echo "Erro: ", $e;
}

// Feito em:
// 08/12/2024